### PR TITLE
TLT-3795 Python 3 and Django 2 upgrade

### PR DESCRIPTION
**Pull request by Chris-Thornton-Harvard**
Upgrades this project to run with Python 3 and Django 2

### CODE REVIEW

**Comment by cmurtaugh** [comment]
Looks like this line needs to be added to `aws.py`: 
```
ALLOWED_CIDR_NETS = [SECURE_SETTINGS.get('vpc_cidr_block')]
```

**Comment by Chris-Thornton-Harvard** [comment]
I have that line in base.py , should it only be in aws.py?

**Comment by cmurtaugh** [comment]
Oh, I missed it! It should be ok for it to be in either place. I noticed this error in dev which made me think it wasn’t set: 

ERROR 2020-05-12T12:30:36.310Z django.security.DisallowedHost:77 Invalid HTTP_HOST header: '10.39.83.5:8000'. You may need to add '10.39.83.5' to ALLOWED_HOSTS.

File: canvas_course_info/settings/base.py [hunk 1]
```diff
@@ -35,17 +32,16 @@
     'django.contrib.staticfiles',
     'course_info',
     'icommons_ui',
-)
+]
 
-MIDDLEWARE_CLASSES = (
+MIDDLEWARE = [
```
Comment (cmurtaugh) [inline canvas_course_info/settings/base.py:37]: I see why the `ALLOWED_CIDR_NETS` setting isn't having an effect -- this needs to be added to the top of `MIDDLEWARE`: 
```
    'allow_cidr.middleware.AllowCIDRMiddleware',
```

**Comment by cmurtaugh** [comment]
:+1:

**Pull request merged**